src/os.rs

```diff
@@ -186,7 +186,7 @@ impl ReadRng {
           not(target_os = "freebsd"),
           not(target_os = "fuchsia"),
           not(target_os = "ios"),
-          not(target_os = "nacl"),
```

Comment:
This is removed because `nacl` is dead? Seems reasonable to me, but it is not mentioned in the commit message.

Reply:
Sure, my only point was that this is not mentioned in the commit message.

Reply:
Then you have a good point :smile:. But the actual commit that removed `nacl` was two months ago (this is just one line that was forgotten), so not much to do here.